### ISSUE TITLE
Ranks for metrics whose optimum is 0 like mean_absolute_error is in reverse order

### ISSUE THREAD

**Issue by rabsr** [automl/auto-sklearn#1078]
## Describe the bug ##
There are various metrics whose optimum value is 0 and for them greater the value, worst are the results. But this is not considered while calculating ranks in cv_results. While calculating ranks, it is always considered that greater the value, better the results. 

## To Reproduce ##
Steps to reproduce the behaviour:
Run any regression problem with metric mean_absolute_error.

Results from cv_results_
```
>>> cv_results_[["mean_test_score","rank_test_scores", "status"]]
   mean_test_score  rank_test_scores   status
0     2.354041e+00                 6  Success
1     2.649970e+00                 4  Success
2     2.758945e+00                 3  Success
3     2.147484e+09                 1  Timeout
4     2.581370e+00                 5  Success
5     2.269768e+00                 7  Success
6     2.147484e+09                 1  Timeout
```

## Expected Result ##
```
>>> cv_results_[["mean_test_score","rank_test_scores", "status"]]
   mean_test_score  rank_test_scores   status
0     2.354041e+00                 2  Success
1     2.649970e+00                 4  Success
2     2.758945e+00                 5  Success
3     2.147484e+09                 6  Timeout
4     2.581370e+00                 3  Success
5     2.269768e+00                 1  Success
6     2.147484e+09                 6  Timeout
```

## Environment and installation: ##

Please give details about your installation:

* Is your installation in a virtual environment or conda environment? virtualenviron
* Python version - 3.7
* Auto-sklearn version - From development branch

**Comment by rabsr** [comment]
I am working on a fix and will raise a PR soon.

**Comment by mfeurer** [comment]
Thanks for pointing this out. Could you please have a look at #1075 before starting to work on this or base your work on that? @franchuterivera could you please give an update on the state of #1075?

**Comment by mfeurer** [comment]
Fixed via #1079
